### PR TITLE
[Feature] #23 DictionaryEquipmentContentView's hifi UI

### LINKED ISSUE #23: [Feature] Construct EquipmentDictionaryContentView's Detail Tools UI
# 유저는 상세 장비 콘텐츠를 확인할 수 있다.
- 상세 장비 콘텐츠를 image와 text로 View에 띄워준다

## 🤔 Completion conditions

- [ ] Construct ToolDictionaryContentView's Detail Tools UI like the design of a sketch ( hi-fi )

### LINKED ISSUE #84: [Feature] DictionaryEquipmentContentView Data Binding
# 유저는 상세 장비 콘텐츠를 확인할 수 있다.
- 상세 장비 콘텐츠의 Entity data를 불러와서 바인딩 한다

## 🤔 Completion conditions

- [ ] make ViewModel
- [ ] make Repository and UseCase
- [ ] can fetchImage

### PR DESCRIPTION
**Pull request by rriver2**
# Issue Number 
🔒 Close #23

## Changes

- Changed files
campair/campair/Utility/ViewExtension.swift
campair/campair/View/Dictionary/Content/DictionaryContentView.swift
campair/campair/View/Editor/Detail/EditorDetailView/EditorDetailContentsVersionView.swift

## New features

- add DictionaryEquipmentContentView's hifi UI

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

**Review by sustainable-git — APPROVED** [review]
Finally, we don't use magic number for font!

**Comment by rriver2** [comment]
> Finally, we don't use magic number for font!

Yes ! and Young praised us for following Accessibility!

File: campair/campair/View/Dictionary/Content/DictionaryContentView.swift [hunk 1]
```diff
@@ -8,13 +9,77 @@
...
+                Rectangle()
+                    .frame(height: 8)
+                    .foregroundColor(Color(hex: "FEFCFB"))
+                    .padding(.vertical, 40)
+                if let prosArray = self.equipmentContent.pros {
+                    Text("장점")
+                        .fontWeight(Font.Weight.bold)
+                        .padding(.bottom, 10)
+                        .bodyBold000000Opacity60()
+                    let pros = prosArray.reduce("") {"\($0)\($1)\n"}
```
Comment (Gobans) [inline campair/campair/View/Dictionary/Content/DictionaryContentView.swift:41]: 🫢 High level code. What does it mean? You combined two pros in one line?

**Review by Gobans — APPROVED** [review]
Thanks for your work!! good job

File: campair/campair/View/Dictionary/Content/DictionaryContentView.swift (hunk 1 above)
Comment (rriver2) [reply campair/campair/View/Dictionary/Content/DictionaryContentView.swift:41]: thank... you... check down below.
"의자에서 누워있는 듯한 느낌이에요./n'쉼'을 목적으로 나온 의자./n다리 폭이 넓어서 넘어질 위험이 적어요."

**Pull request merged**